README.rst

```diff
@@ -2,55 +2,133 @@
 Shapely
 =======
 
-|github-actions| |coveralls|
+.. Documentation at RTD — https://readthedocs.org
+
+.. image:: https://readthedocs.org/projects/shapely/badge/?version=stable
+   :alt: Documentation Status
+   :target: https://shapely.readthedocs.io/en/stable/
+
+.. Github Actions status — https://github.com/shapely/shapely/actions
 
 .. |github-actions| image:: https://github.com/shapely/shapely/workflows/Tests/badge.svg?branch=main
+   :alt: Github Actions status
    :target: https://github.com/shapely/shapely/actions?query=branch%3Amain
 
+.. Travis CI status -- https://travis-ci.com
+
+.. image:: https://travis-ci.com/shapely/shapely.svg?branch=main
+   :alt: Travis CI status
+   :target: https://travis-ci.com/github/shapely/shapely
+
+.. PyPI
+
+.. image:: https://img.shields.io/pypi/v/shapely.svg
+   :alt: PyPI
+   :target: https://pypi.org/project/shapely/
+
+.. Anaconda
+
+.. image:: https://img.shields.io/conda/vn/conda-forge/shapely
+   :alt: Anaconda
+   :target: https://anaconda.org/conda-forge/shapely
+
+.. Coverage
+
 .. |coveralls| image:: https://coveralls.io/repos/github/shapely/shapely/badge.svg?branch=main
    :target: https://coveralls.io/github/shapely/shapely?branch=main
 
+.. Zenodo
+
+.. .. image:: https://zenodo.org/badge/191151963.svg
+..   :alt: Zenodo
+..   :target: https://zenodo.org/badge/latestdoi/191151963
+
```

Comment:
This DOI is for pygeos. I've never set these up, but we'd probably need to set one up for Shapely.

Reply:
Yeah, indeed (that's the reason that I left it commented out for now). Thanks for opening the issue about it.

Reply:
xref #1455 as a starting point